iotdb-core/consensus/src/test/java/org/apache/iotdb/consensus/ratis/RecoverReadTest.java

```diff
@@ -233,7 +233,7 @@ public void consistentReadTimeout() throws Exception {
     // wait until active leader to serve read index requests
     miniCluster.waitUntilActiveLeaderElected();
 
-    // query during redo: get exception that ratis is under recovery
-    Assert.assertThrows(RatisReadUnavailableException.class, () -> miniCluster.readThrough(0));
+    // query, the result will return since it will be queued and handled by leader once ready
+    Assert.assertEquals(50, miniCluster.mustRead(0));
   }
```

Comment:
Since now read-index can queue up during leader not ready time, this read will successfully return, instead of throwing out an exception.